cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -180,6 +216,7 @@ def fn(rows):
 
         return fn
 
+    # Repartition the DataFrame, sort and map it, return as a DataFrame.     
```

Comment:
Sort and map what is the question. Can you rephrase this line? (please try to understand what `repartition`, `sortWithinPartitions`, etc. are.)

Reply:
# Rearrange the rows in the DataFrame based on the partitioning expression.
# Sort every partition in the DataFrame based on specific columns.
# Apply the 'add_elapsed_column' method to each partition in the dataset, and convert it to a DataFrame.
What do you think?

Reply:
# Rearrange the rows in the DataFrame based on the partitioning expression.
# Sort every partition in the DataFrame based on specific columns.
# Apply the 'add_elapsed_column' method to each partition in the dataset, and convert it to a DataFrame.
What do you think?

Reply:
"Rearrange the rows in the DataFrame based on the partitioning expression.
Sort every partition in the DataFrame based on specific columns.
Apply the 'add_elapsed_column' method to each partition in the dataset, and convert it to a DataFrame."
What do you think?